shell/plugins/steve/caches/resourceCache.js

```diff
@@ -122,6 +139,7 @@ export default class ResourceCache {
       });
     }
 
+    // TODO: RC why are we filtering here? shouldn't the cache be 1-1 with all / namespace / selector?
```

Comment:
If the cache is currently watching everything and I get a request for a specific selector and/or namespace then there's no reason to make a request to the API when I can just filter the full cache, this'll also be combined with the searches that the request will pass in when pr4 is merged in.

Reply:
i've added force to this and fixed a few cases (mainly none, all --> some --> all where we were setting the current value to something restricted when we were only filtering)